### ISSUE TITLE
Accessibility issues for video

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#1638]

**Comment by meols** [comment]
Is it possible to define issues raised in Siteimprove for video within the video component(s)? ![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/47c98b5d-8da7-44f4-a5e1-cc6943722f93)

**Comment by padms** [comment]
@meols The issues with the muted videos can be marked as non issue ( it can exist with out captions ) in siteimprove. Here is a video that shows how to review this issue https://help.siteimprove.com/support/solutions/articles/80000448552-accessibility-reviewing-potential-issues

**Comment by padms** [comment]
@meols The issues with the muted videos can be marked as non issue ( it can exist with out captions ) in siteimprove. Here is a video that shows how to review this issue https://help.siteimprove.com/support/solutions/articles/80000448552-accessibility-reviewing-potential-issues

**Comment by meols** [comment]
Thanks @padms - this is what we are doing today. But since this issue will pop up every time we add a new video (is my understanding at least) I was wondering if (some of) it could be handled in the components.

**Comment by fernandolucchesi** [comment]
> Thanks @padms - this is what we are doing today. But since this issue will pop up every time we add a new video (is my understanding at least) I was wondering if (some of) it could be handled in the components.

We can add a "description" field in the video component to assist with accessibility. How does it sound?

**Comment by fernandolucchesi** [comment]
@meols

**Comment by meols** [comment]
@fernandolucchesi - what you suggest is information for the editors, right - not end users? What I am thinking about is an accessibility proofed way of editing the video component to "tell" crawlers and screenreaders that a video is purely decorative and e.g. doesn't have caption (and other flagged issues) - if possible? The goal is to make it better/more understandable for end users with challenges, not to do a fix/explain for editors.

**Comment by SvSven** [comment]
From what I understand you can't "mark" a video as purely decorative like you can do with images. I think these might just always pop up as a reminder/checklist since I don't _think_ there is a standard way of handling all accessibility things surrounding videos yet; so it might be that SiteImprove can't always know. That being said, I'm not an expert on accessibility with videos

**Comment by SvSven** [comment]
@NickHaggerty1 @meols 

I should note btw that in accordance with the WCAG guidelines, we always have to display video controls; and that this is something uutilsynet also tests on. This means that our current setup for the video header, and video players where editors can choose to hide the video controls, are not in compliance.

> ### Success criterion 2.2.2 Pause, stop, hide
>
> #### Purpose of 2.2.2
>
> Moving content, such as video, animations, real-time games, automatic updates, etc., may lead to users being unable to concentrate on other content on the page.
>
> It must therefore be possible to stop, pause or hide moving content, or to control the update frequency. Content that has been paused may either continue in real time or start from the point at which the content was paused.

![image.png](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/d5f2b7ef-3741-45e2-9ccb-126d122a92fb)

See: https://www.uutilsynet.no/english/interpretation-and-overview-test-procedures-wcag-20-and-aa/138#51_principle_perceivable

**Comment by millianapia** [comment]
@NickHaggerty1 this is ready to be reviewed

**Comment by millianapia** [comment]
made a small change regarding the logic of how the button is placed. Also changed the size of the button based on screen-size based on feedback from @NickHaggerty1. This is in need for a new round of code review @fernandolucchesi @padms

**Comment by millianapia** [comment]
@NickHaggerty1 we did some refactoring and i added some accessibility labels on the videoplayer button, so would be great with a last check regarding this issue

**Comment by NickHaggerty1** [comment]
@millianapia Hey, not sure how to test this? Am I supposed to see labels on hover? or?
